sparse/_common.py

```diff
@@ -33,6 +36,23 @@
 )
 
 
+@numba.njit
+def nan_check(*args):
+    for i in literal_unroll(args):
+        if np.isnan(np.min(np.asarray(i))):
+            return True
+    return False
+
+
+def check_class_nan(test):
+    if isinstance(test, SparseArray):
+        return nan_check(test.data, test.fill_value)
```

Comment:
```suggestion
        return nan_check(test.fill_value, test.data)
```

The check may be faster this way.

Reply:
So does this one.